posts/2021/04/a-step-towards-educating-with-Spyder.md

```diff
@@ -0,0 +1,89 @@
+<!--
+.. title: A step towards educating with Spyder 
+.. slug: a-step-towards-educating-with-Spyder
+.. date: 2021-04-11 08:00:00 UTC-06:00
+.. author: Juanita Gomez
+.. tags:
+.. category:
+.. link:
+.. description:
+.. type: text
+-->
+
+
+As a community manager in the Spyder team, I have been looking for ways of 
+involving more users in the community and making Spyder useful for a larger 
+number of people. With this, a new idea came: Education.
+
+For the past months, we have been wondering with the team whether Spyder 
+could also serve as a teaching-learning platform, especially in this era 
+where remote instruction has become necessary. We submitted a proposal to the 
+Essential Open Source Software for Science (EOSS) program of the Chan 
+Zuckerberg Initiative, during its third cycle, with the idea of providing a 
+simple way inside Spyder to create and share interactive tutorials on topics 
+relevant to scientific research. Unfortunately, we didn’t get this funding, 
+but we didn’t let this great idea die.
+
+<!-- TEASER_END -->
+
+We submitted a second proposal to the [Python Software Foundation](https://www.python.org/psf/) 
+from which we were awarded $4000. For me, this is the perfect opportunity for 
+us to take the first step towards using Spyder for education.
+
+## What the project is about
+
+The goal of this project is to create specialized Python online training 
+content that uses Spyder as the main platform to deliver it. The grant will 
+cover the development of three practical workshops:
+
+1. Python for Financial Data Analysis with Spyder
+2. Python for Scientific Computing and Visualization with Spyder
+3. Spyder 5 Plugin Development
+ 
+They will be included as part of [Spyder’s documentation](https://docs.spyder-ide.org/current/index.html) 
+for remote learning, but they will also be used as hands-on materials for talks and workshops. 
+
+These materials are meant for users to learn how Spyder can accelerate their 
+workflow when working with Python in scientific research and data analysis. 
+The idea is for us to provide a way in which we can help people get the most 
+out of Spyder by applying it in their day-to-day jobs.
+ 
+The first two workshops will cover aspects such as data exploration and 
+visualization with Spyder’s variable explorer and plots panes, getting 
+documentation through Spyder’s help pane, writing good quality and efficient 
+code using Spyder’s code analysis and profiler, etc.
+
+Our last workshop will demonstrate how to create a plugin for Spyder, which, 
+thanks to our new API in [Spyder 5](https://github.com/spyder-ide/spyder/releases/tag/v5.0.0), 
+released in the last few days, will allow users to easily customize and 
```

Comment:
I'd mention either the date of the release, or "released in April 2021".

Reply:
Agreed. I added some tags and fixed this part. Thanks!